src/components/Context.ts

```diff
@@ -15,13 +15,17 @@ export interface ReactReduxContextValue<
   noopCheck: CheckFrequency
 }
 
-let realContext: Context<ReactReduxContextValue> | null = null
+const ContextKey = Symbol.for(`react-redux-context-${ReactVersion}`)
+const gT = globalThis as { [ContextKey]?: Context<ReactReduxContextValue> }
```

Comment:
`globalThis` is not present in Android webview version < 71. I faced issues in Android OS version <= 8.1.0 due to this.